dash-renderer/tests/isAppReady.test.js

```diff
@@ -20,34 +22,38 @@ describe('isAppReady', () => {
 
     it('executes if app is ready', async () => {
         let done = false;
-        Promise.resolve(isAppReady(
-            [{ namespace: '__components', type: 'b', props: { id: 'comp1' } }],
-            { strs: { comp1: [0] }, objs: {}, events: emitter },
-            ['comp1']
-        )).then(() => {
-            done = true
+        Promise.resolve(
+            isAppReady(
+                [{namespace: '__components', type: 'b', props: {id: 'comp1'}}],
+                {strs: {comp1: [0]}, objs: {}, events: emitter},
+                ['comp1']
+            )
+        ).then(() => {
+            done = true;
         });
 
         await new Promise(r => setTimeout(r, WAIT));
-        expect(done).toEqual(true);
+        expect(done).to.equal(true);
```

Comment:
Slight syntax differences with `toEqual` and `toBe` being replaced by `to.equal` and `to.deep.equal` depending on the situation